libs/components/tabs/testing/src/modules/sectioned-form/sectioned-form-harness.ts

```diff
@@ -0,0 +1,109 @@
+import { HarnessPredicate } from '@angular/cdk/testing';
+import { SkyComponentHarness } from '@skyux/core/testing';
+
+import { SkySectionedFormHarnessFilters } from './sectioned-form-harness-filters';
+import { SkySectionedFormSectionContentHarness } from './sectioned-form-section-content-harness';
+import { SkySectionedFormSectionHarness } from './sectioned-form-section-harness';
+import { SkySectionedFormSectionHarnessFilters } from './sectioned-form-section-harness-filters';
+
+/**
+ * Harness for interacting with a sectioned form component in tests.
+ */
+export class SkySectionedFormHarness extends SkyComponentHarness {
+  /**
+   * @internal
+   */
+  public static hostSelector = 'sky-sectioned-form';
+
+  #getTabs = this.locatorForOptional('div.sky-sectioned-form-tabs');
+  #getContent = this.locatorForOptional('div.sky-sectioned-form-content');
+
+  /**
+   * Gets a `HarnessPredicate` that can be used to search for a
+   * `SkySectionedFormHarness` that meets certain criteria.
+   */
+  public static with(
+    filters: SkySectionedFormHarnessFilters,
+  ): HarnessPredicate<SkySectionedFormHarness> {
+    return SkySectionedFormHarness.getDataSkyIdPredicate(filters);
+  }
+
+  /**
+   * Gets the `SkySectionedFormSectionHarness` for the currently active section.
+   */
+  public async getActiveSection(): Promise<
+    SkySectionedFormSectionHarness | undefined
+  > {
+    const sections = await this.getSections();
+
+    if (sections) {
+      for (const section of sections) {
+        if (await section.isActive()) {
+          return section;
+        }
+      }
+    }
+
+    return undefined;
+  }
+
+  public async getActiveSectionContent(): Promise<
+    SkySectionedFormSectionContentHarness | undefined
+  > {
+    if (await this.#getContent()) {
+      const contents = await this.locatorForAll(
+        SkySectionedFormSectionContentHarness,
+      )();
+      for (const content of contents) {
+        if (await content.isVisible()) {
```

Comment:
so looking at the code, it looks like content is not loaded until the tab is activated for the first time. Since this changes depending on if consumers set `maintainSectionContent` to true or not. Wouldnt it make sense to get the active section from `getActiveSection()` and then querying the active content based on its `aria-controls` value?

Reply:
from conversation: in mobile view when tabs are hidden, there is no way for us to get the tab's `aria-controls` and would have to fall back to this implementation anyways.